examples/java/gradle-shared-module/module-b/cdktf.json

```diff
@@ -0,0 +1,11 @@
+{
+  "language": "java",
+  "app": "../gradlew -q :module-b:run",
+  "output": "./",
```

Comment:
```suggestion
  "//": "The target output directory is set to the current working directory so that definitions generated by cdk can be referenced in local HCL sources"
  "output": "./",
```